src/backend/distributed/worker/worker_data_fetch_protocol.c

```diff
@@ -482,6 +482,7 @@ worker_apply_sequence_command(PG_FUNCTION_ARGS)
 	/* run the CREATE SEQUENCE command */
 	ProcessUtility(commandNode, commandString, PROCESS_UTILITY_TOPLEVEL,
 				   NULL, None_Receiver, NULL);
+	CommandCounterIncrement();
```

Comment:
Read the commit message; basically we create a sequence and then expect it to be visible. In PostgreSQL 10 it isn't yet, and there could be two reasons why:

  * We made some other call in Citus that pre-PostgreSQL 10 called CCI but no longer does, implicitly relying on command counter increment to make our change visible
  * Due to [changes in how PostgreSQL's stores and accesses sequence attributes](https://www.postgresql.org/message-id/flat/CAB7nPqSJB1h535AcQ%2BZOyqM0Sg6VORf0hL_Vrw96uvajR03hQg%40mail.gmail.com#CAB7nPqSJB1h535AcQ+ZOyqM0Sg6VORf0hL_Vrw96uvajR03hQg@mail.gmail.com), the newly created sequence was previously visible, but now that we have to access it through `SearchSysCache` we must increment the command counter to see it